index.html

```diff
@@ -810,14 +810,14 @@ <h3>Proofs</h3>
 
           <dt><dfn class="lint-ignore">proofValue</dfn></dt>
           <dd>
-A string value that contains the base-encoded binary data necessary to verify
-the digital proof using the `verificationMethod` specified. The contents of the
-value MUST be expressed with a header and encoding as described in Section
-<a data-cite="CONTROLLER-DOCUMENT#multibase-0">2.4 Multibase</a> of
-the [[[CONTROLLER-DOCUMENT]]] specification. The contents of this value are
-determined by a specific cryptosuite and set to the <em>proof value</em>
-generated by the <a href="#add-proof">Add Proof Algorithm</a> for that
-cryptosuite. Alternative properties with different encodings specified by the
+A string value that expresses base-encoded binary data necessary to verify the 
+digital proof using the verificationMethod specified. The value MUST use a
```

Comment:
I think the backticks got dropped accidentally here:

```suggestion
digital proof using the `verificationMethod` specified. The value MUST use a
```